### PR TITLE
bump lodash to version to 4.11.1

### PR DESCRIPTION
**Pull request by RickButler**
Didn't see a PR for this following the bump to lodash 4.11.1 in the other ampersand repositories.

### CODE REVIEW

**Comment by cdaringe** [comment]
thanks @RickButler!  looks good to me!

File: ampersand-form-view.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /*$AMPERSAND_VERSION*/
 var View = require('ampersand-view');
-var set = require('lodash.set');
-var isFunction = require('lodash.isfunction');
-var result = require('lodash.result');
+var set = require('lodash/set');
+var isFunction = require('lodash/isfunction');
```
Comment (cdaringe) [inline ampersand-form-view.js]: actually, this is think is supposed to be `isFunction`. did you run the tests?

Comment (cdaringe) [inline ampersand-form-view.js]: case matters in some envs

Comment (pgilad) [inline ampersand-form-view.js]: *Linux*

Comment (RickButler) [inline ampersand-form-view.js]: I did run the tests, I was on my windows machine at the time =(

**Comment by RickButler** [comment]
@cdaringe @pgilad fixed the casing and squashed the commit to make everything pretty.

**Comment by cdaringe** [comment]
🎲 🎲 🎲 🎲 🎲 
^^ yahtzee

one more req'd +1.

**Comment by pgilad** [comment]
:+1:

**Pull request merged**